packages/components/src/StatusLabel/StatusLabel.css

```diff
@@ -1,20 +1,48 @@
 :root {
-  --statusLabel-icon-diameter: 10px;
+  --labelBackgroundColor: var(--color-success--surface);
+}
+
+.success {
```

Comment:
Not sure the best way to do this. Success is the default, but something was getting mad at me because success was being passed as a class as a status but was included.

Reply:
I think this is great! I don't see anything getting mad?